spec/collator.html

```diff
@@ -49,46 +49,39 @@
 
       <emu-alg>
         1. If collator has an [[initializedIntlObject]] internal slot with value *true*, throw a *TypeError* exception.
```

Comment:
Unrelated to this PR, "collator" should be in italics (throughout this entire algorithm)